src/appMain/daemon.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+cd $(dirname $0)
+COMMAND=$1
+EXTERNAL_POLICIES=$2
+DIR=$(pwd)
+
+CORE_PID_FILE=${DIR}/core.pid
+CORE_APPLICATION_NAME=smartDeviceLinkCore
+PM_PID_FILE=${DIR}/policy_manager.pid
+PM_APPLICATION_NAME=sample_policy_manager.py
+
+function core_start() {
+  if [ -f "$CORE_PID_FILE" ] && [ -n "$(ps -p $(cat $CORE_PID_FILE) -o pid=)" ]; then
+    echo "Core is already running"
+    return 1
+  elif [ -n "$(pgrep -f $CORE_APPLICATION_NAME)" ]; then
+    echo "Core is already running outside of this script"
+    echo "All instances of Core can be stopped using the \"kill\" command"
+    return 2
+  else
+    echo "Starting SmartDeviceLink Core"
+    LD_LIBRARY_PATH=$DIR ${DIR}/${CORE_APPLICATION_NAME} &
+    CORE_PID=$!
+    echo $CORE_PID > $CORE_PID_FILE
+    return 0
+  fi
+}
+
+function core_stop() {
+  RESULT=1
+  if [ -f "$CORE_PID_FILE" ] && [ -n "$(ps -p $(cat $CORE_PID_FILE) -o pid=)" ]; then
+    echo "Stopping SmartDeviceLink Core"
+    CORE_PID=$(cat $CORE_PID_FILE)
+    kill $CORE_PID
+
+    # If Core doesn't close normally within 10 seconds, force-close it
+    timeout 10s tail --pid=$CORE_PID -f /dev/null
+    if [ -n "$(ps -p $CORE_PID -o pid=)" ]; then
+      echo "Core did not shut down properly, force-killing"
+      kill -9 $CORE_PID
+    fi
+    RESULT=0
+  fi
+
+  if [ -f "$CORE_PID_FILE" ]; then
+    rm $CORE_PID_FILE
+  fi 
+  return $RESULT
+}
+
+function pm_install_dependencies() {
+  pip3 list | grep -F tornado > /dev/null
+  if [ $? -eq 1 ]; then
+    echo "Installing tornado python package"
+    sudo -H pip3 install tornado
```

Comment:
Would be nice to specify tornado in dependencies, but not force installation during sdl_run